### PR TITLE
Fix for rpm/deb installation errors

### PR DESCRIPTION
**Pull request by vinodtalati**
See issue [82](https://github.com/intuit/wasabi/issues/82)
This is a partial fix for wasabi-main-*.rpm  (or .deb)

Will try and add fixes here for wasabi-ui-*.rpm (or .deb)

### CODE REVIEW

**Comment by vinodtalati** [comment]
Tested on Ubuntu and looks fine. No more installation errors :)


```
vinod@vinod-virtual-machine:~/Desktop$ sudo dpkg -i wasabi-main-build_1.0.20160902232623-SNAPSHOT-20160915233846_all.deb 
Selecting previously unselected package wasabi-main-build.
(Reading database ... 172243 files and directories currently installed.)
Preparing to unpack wasabi-main-build_1.0.20160902232623-SNAPSHOT-20160915233846_all.deb ...
Unpacking wasabi-main-build (1.0.20160902232623-SNAPSHOT-20160915233846) ...
Setting up wasabi-main-build (1.0.20160902232623-SNAPSHOT-20160915233846) ...
vinod@vinod-virtual-machine:~/Desktop$ 
vinod@vinod-virtual-machine:~/Desktop$ 
vinod@vinod-virtual-machine:~/Desktop$ sudo dpkg -i wasabi-ui-build_1.0.20160902232623-SNAPSHOT-20160915234110_all.deb 
Selecting previously unselected package wasabi-ui-build.
(Reading database ... 172324 files and directories currently installed.)
Preparing to unpack wasabi-ui-build_1.0.20160902232623-SNAPSHOT-20160915234110_all.deb ...
Unpacking wasabi-ui-build (1.0.20160902232623-SNAPSHOT-20160915234110) ...
Setting up wasabi-ui-build (1.0.20160902232623-SNAPSHOT-20160915234110) ...
vinod@vinod-virtual-machine:~/Desktop$
```

File: bin/wasabi.sh [hunk 1]
```diff
@@ -255,9 +255,9 @@ package() {
   version=$(fromPom . build project.version)
   # FIXME: server ip
   server="http://localhost:8080"
-  home=$(fromPom . build application.home)
+  home=$(fromPom ./modules/main build application.home)
```
Comment (shoeffner) [inline bin/wasabi.sh]: Oh, that must have slipped through while we removed the python dependencies, great catch!

**Comment by jwtodd** [comment]
looks good to me ... heck it looks great :)

others are reviewing/verifying atm so i defer to them.

but this looks good ... odd we have to add the "null object or invalid expression" conditional ... but we if we have to we have to.

**Comment by aqua0ps** [comment]
verified and it works on my single node terraform installation in aws :shipit:

**Pull request merged**